types/plop/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for plop 1.8
+// Project: https://github.com/amwmedia/plop
+// Definitions by: Alex Young <https://github.com/alsiola>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import { Inquirer, Question, PromptModule } from "inquirer";
+
+declare module "plop" {
+    interface Action {
+        type: string;
+        path: string;
+        templateFile: string;
+    }
+
+    interface GeneratorConfig {
+        description?: string;
+        prompts: ReadonlyArray<Question>;
+        actions: ReadonlyArray<Action>;
+    }
+
+    interface GeneratorListItem {
+        name: string;
+        description?: string;
+    }
+
+    interface Plop {
+        inquirer: Inquirer;
+        handlebars: object;
+        registerHelper(name: string, fn: (...args: any[]) => any): void;
```

Comment:
You would know better than me, but from a brief glance at the `plop` API, it seems like this is called `addHelper`?